src/Microsoft.Azure.WebJobs.Host/Executors/FunctionInvoker.cs

```diff
@@ -12,6 +12,7 @@ internal class FunctionInvoker<TReflected, TReturnValue> : IFunctionInvoker
         private readonly IReadOnlyList<string> _parameterNames;
         private readonly IFactory<TReflected> _instanceFactory;
         private readonly IMethodInvoker<TReflected, TReturnValue> _methodInvoker;
+        private TReflected _instance;
```

Comment:
>private TReflected _instance; [](start = 8, length = 29)

We should not be caching this.  It breaks semantics about how many times IJobActivator is called